sdk/extensions/chronopost/karrio/providers/chronopost/tracking.py

```diff
@@ -0,0 +1,84 @@
+import typing
+from urllib import response
+import karrio.lib as lib
+import karrio.core.models as models
+import karrio.providers.chronopost.utils as provider_utils
+import karrio.providers.chronopost.error as provider_error
+import chronopost_lib.trackingservice as chronopost
+
+
+def parse_tracking_response(
+    responses: typing.List[lib.Element],
+    settings: provider_utils.Settings,
+) -> typing.Tuple[typing.List[models.TrackingDetails], typing.List[models.Message]]:
+    tracking_nodes: typing.List[chronopost.eventInfoComp] = [
+        lib.find_element("events", response, chronopost.eventInfoComp)
+        for response in responses
+    ]
+    errors = [
+        provider_error.parse_error_response(result, settings) for result in responses
+    ]
```

Comment:
You don't need the wrapping list. since it will the parse error function will retrieve the list of errors
```suggestion
    errors = provider_error.parse_error_response(result, settings)
```